src/models/serverless.ts

```diff
@@ -50,6 +50,7 @@ export interface ServerlessAzureProvider {
   hostingEnvironment?: ResourceConfig;
   virtualNetwork?: ResourceConfig;
   armTemplate?: ArmTemplateConfig;
+  runtime: string; 
```

Comment:
Does this need to be a required parameter?

Reply:
Yes it is a required parameter based on the discussion